### ISSUE TITLE
Weighted cross entropy loss

### ISSUE THREAD

**Issue by champagne-yellow** [open-mmlab/mmdetection#8668]
If I want to weight each class on the cross entropy loss function, what is the shape of the weights？

**Comment by hhaAndroid** [comment]
@champagne-yellow (C, )
